app/main.py

```
@@ -0,0 +1,54 @@
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from fastapi.responses import JSONResponse
+
+# FastAPIアプリケーションの初期化
+app = FastAPI(
+    title="Jockey Data API",
+    description="騎手IDに基づいてS3からpickleファイルを取得し、JSON形式で返却するAPI",
+    version="0.1.0",
+)
+
+# CORSミドルウェアの設定（全オリジンを許可）
```

Ruff (0.13.3)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)